readme.txt

```diff
@@ -54,6 +68,16 @@ The four phases of the project are Editing, Customization, Collaboration, and Mu
 - <a href="https://wordpress.org/gutenberg/handbook/">Documentation: Creating Blocks, Reference, and Guidelines</a>
 - <a href="https://wordpress.org/gutenberg/handbook/reference/faq/">Additional frequently asked questions</a>
 
+= What’s with all the 1-star reviews? =
+
+Many reviews are from plugin installers who might expect a bug-free, well-polished editor, that doesn’t change often. However, Gutenberg is a continually evolving work in progress, subject to frequent changes (typically every one to two weeks!), some of which can be surprising to users — and admittedly, may break non-stable functionality. This is why the plugin is not recommended for production sites.
+
+We do read every review, but the best way to provide feedback is through <a href="https://github.com/WordPress/gutenberg/issues">the Gutenberg GitHub issues page</a>. And new contributors are always welcome to join the design and development effort!
+
+= How can I get the old “classic” editor back? =
```

Comment:
I find this strange to include as well and would recommend removing. It's precious real estate on this plugin page and mixing up Gutenberg with the Classic Editor plugin feels like we're not properly preparing folks for the future of WordPress.

Reply:
💯  The target audience of the plugin is more early adopters than a late majority or laggards, so this section seems counterintuitive for people that want to try the latest features and prepare for the future. I would also recommend removing it.